aidants_connect_web/tests/test_functional/test_view_mandats.py

```diff
@@ -5,15 +5,15 @@
 
 from aidants_connect_web.tests.factories import (
     AidantFactory,
-    MandatFactory,
+    AutorisationFactory,
     UsagerFactory,
 )
 from aidants_connect_web.tests.test_functional.testcases import FunctionalTestCase
 from aidants_connect_web.tests.test_functional.utilities import login_aidant
 
 
 @tag("functional")
-class ViewMandats(FunctionalTestCase):
+class ViewMandatsTestCase(FunctionalTestCase):
```

Comment:
Idem question sur l'ajout de TestCase
Est-ce que ça ne serait pas plutôt `ViewAutorisations` ?

Reply:
> Est-ce que ça ne serait pas plutôt `ViewAutorisations` ?

Toujours dans l'optique de laisser le vocable `mandat` là où cela fait/fera sens dans la version cible, j'ai l'impression que là c'est bien "la collection" qu'on voit ? Pas l'ombre d'un souci pour modifier ça, c'est une question de "vision produit" et je pense que tu es très très probablement plus au clair que moi de ce point de vue 😁

Reply:
```suggestion
class ViewAutorisationsTest(FunctionalTestCase):
```
Pour être iso avec l'expérience utilisateur -> on voir bien les autorisations et non les collections d'autorisations.